From c780beab594c3fb0e77b0d5b3c9461b146f3e269 Mon Sep 17 00:00:00 2001
From: Artem Pelenitsyn <a.pelenitsyn@gmail.com>
Date: Wed, 20 Aug 2025 10:19:20 -0400
Subject: [PATCH] CI: validate.yml: add ghc 9.14-alpha2

- Update ghc-latest.config: 9.12 -> 9.14
- ghc-latest.config: allow-newer:
  - binary
  - containers
  - ghc-bignum
---
 .github/workflows/validate.yml  | 3 ++-
 project-cabal/ghc-latest.config | 4 ++--
 2 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/project-cabal/ghc-latest.config b/project-cabal/ghc-latest.config
index 96042ad58..5e4dd4e3a 100644
--- a/project-cabal/ghc-latest.config
+++ b/project-cabal/ghc-latest.config
@@ -8,11 +8,11 @@
 
 -- NOTE: don't forget to update the compiler version in the conditional
 -- when upgrading to a newer GHC
-if impl(ghc >= 9.12.0)
+if impl(ghc >= 9.14.0)
     allow-newer:
     --windns:*, rere:*, tree-diff:*, uuid-types:*, these:*, assoc:*, semialign:*, indexed-traversable-instances:*, indexed-traversable:*, OneTuple:*, scientific:*, time-compat:*, text-short:*, integer-conversion:*, generically:*, data-fix:*, binary:*
     -- Artem, 2024-04-21: I started and then gave up...
-        *:base, *:template-haskell, text-short, *:deepseq, *:bytestring, *:ghc-prim
+        *:base, *:template-haskell, text-short, *:deepseq, *:bytestring, *:ghc-prim, *:ghc-bignum, *:containers
 
 --    repository head.hackage.ghc.haskell.org
 --        url: https://ghc.gitlab.haskell.org/head.hackage/
-- 
2.43.0

